namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/elasticache.tf

```
@@ -0,0 +1,37 @@
+################################################################################
+# Interventions Elasticache for ReDiS
+################################################################################
+
+module "hmpps_interventions_elasticache_redis" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=4.2"
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  application            = var.application
+  environment-name       = var.environment-name
+  is-production          = var.is_production
+  infrastructure-support = var.infrastructure_support
+  team_name              = var.team_name
+  number_cache_clusters  = var.number_cache_clusters
+  node_type              = "cache.t2.small"
+  engine_version         = "4.0.10"
+  parameter_group_name   = "default.redis4.0"
+  namespace              = var.namespace
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "hmpps_interventions_elasticache_redis" {
+  metadata {
+    name      = "hmpps-book-video-link-elasticache-redis"
```

This is the secret's name in the namespace, what do you think about
```suggestion
    name      = "ui-session-store-redis"
```

---

oops copy paste error there, good spot! 👀

---

i think i'd like to keep the secret name generic, since it's remotely possible that other interventions services could use the elasticache instance for something other than a session store.